src/headers.toit

```diff
@@ -77,12 +85,22 @@ class Headers:
     write_to buffer
     return buffer.to_string
 
+  /**
+  Creates a copy of this instance.
+  */
+  copy -> Headers:
+    result := Headers
+    headers_.do: | key values/List |
+      copied := List values.size: values[it]
+      result.headers_[key] = copied
+    return result
+
   // Camel-case a string.  Only works for ASCII in accordance with the HTTP
   // standard.  If the string is already camel cased (the norm) then no
   // allocation occurs.
   ascii_normalize_ str:
     alpha := false  // Was the previous character an alphabetic (ASCII) letter.
-    ba := null  // Allocate byte array later if needed.
+    ba /ByteArray? := null  // Allocate byte array later if needed.
```

Comment:
I would call this buffer or something else. Calling it `ba` smells too much like Go to me.

Also, I'd remove the space before `/`.

Reply:
done.